src/Chronos.php

```diff
@@ -268,18 +268,18 @@ protected function createNative(
         }
 
         $relative = static::hasRelativeKeywords($time);
-        if (!empty($time) && $time !== 'now' && !$relative) {
+        if ($time && $time !== 'now' && !$relative) {
             return new DateTimeImmutable($time, $timezone);
         }
 
         $testNow = clone $testNow;
-        $relativetime = self::isTimeExpression($time);
-        if (!$relativetime && $timezone !== $testNow->getTimezone()) {
+        $relativeTime = self::isTimeExpression($time);
+        if (!$relativeTime && $timezone !== $testNow->getTimezone()) {
             $testNow = $testNow->setTimezone($timezone ?? date_default_timezone_get());
         }
 
         if ($relative) {
-            $testNow = $testNow->modify($time);
+            $testNow = $testNow->modify($time ?? 'now');
```

Comment:
We can probably just return the clone if $time is not set, but can look into that later.